QuizWhois.Domain/Services/Implementations/PackService.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using QuizWhois.Common.Models;
+using QuizWhois.Domain.Database;
+using QuizWhois.Domain.Entity;
+using QuizWhois.Domain.Services.Interfaces;
+
+namespace QuizWhois.Domain.Services.Implementations
+{
+    public class PackService : IPackService
+    {
+        private readonly ApplicationContext _dbContext;
+        private readonly ILogger<PackService> _logger;
+
+        public PackService(ApplicationContext context, ILogger<PackService> logger)
+        {
+            _dbContext = context;
+            _logger = logger;
+        }
+
+        public async Task<PackModelResponse> CreatePack(PackModelRequest packModel)
+        {
+            if (packModel == null || packModel.IsDraft == null)
```

Comment:
nullable types have "HasValue" property. it is overall a bit confusing to see boolean compared to null
```suggestion
            if (packModel == null || !packModel.IsDraft.HasValue)
```